src/plugins/irc-events/list.js

```diff
@@ -23,7 +23,9 @@ module.exports = function(irc, network) {
 	irc.on("channel list end", function() {
 		updateListStatus(new Msg({
 			type: "channel_list",
-			channels: network.chanCache.slice(0, MAX_CHANS)
+			channels: network.chanCache.sort(function(a, b) {
+				return a.num_users === b.num_users ? 0 : a.num_users > b.num_users ? -1 : 1;
+			}).slice(0, MAX_CHANS)
```

Comment:
[Per MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Description), the following would work too:

```js
network.chanCache.sort((a, b) => b.num_users - a.num_users).slice(0, MAX_CHANS)
```

Could you give this a shot please?